.github/workflows/deploy_android_production_build.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy Android Production Builds To Firebase
+
+# SECRETS needed:
+### FIREBASE_TOKEN
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  deploy_android_production_to_firebase:
+    name: Deploy android production to Firebase
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Java JDK
+        uses: actions/setup-java@v2.1.0
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Google Services for Staging
+        env:
+          ANDROID_GOOGLE_SERVICE_INFO: ${{ secrets.ANDROID_GOOGLE_SERVICE_INFO }}
+        run: |
+          touch android/google-services.json
+          echo $ANDROID_GOOGLE_SERVICE_INFO > ./android/google-services.json
+
+      - name: Setup Android Signing
+        env:
+          ANDROID_RELEASE_KEYSTORE: ${{ secrets.ANDROID_RELEASE_KEYSTORE }}
+          ANDROID_SIGNING_PROPERTIES: ${{ secrets.ANDROID_SIGNING_PROPERTIES }}
+        run: |
+          touch config/release.keystore
+          echo $ANDROID_RELEASE_KEYSTORE | base64 --decode > config/release.keystore
+          touch signing.properties
+          echo $ANDROID_SIGNING_PROPERTIES | base64 --decode > signing.properties
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - uses: chkfung/android-version-actions@v1.1
+        with:
+          gradlePath: android/build.gradle.kts
+          versionCode: ${{ github.run_number }}
+          versionName: 0.1.0
+
+      - name: Build production release APK
+        run: ./gradlew assembleProductionRelease
+
+      - name: Deploy staging to Firebase
+        uses: wzieba/Firebase-Distribution-Github-Action@v1
+        with:
+          appId: '1:713044215071:android:ef61df1cb34c024e8adc2a'
```

Comment:
I think for appID, it can be public. Actually, based on [this resource](https://firebase.google.com/docs/projects/learn-more#config-files-objects), the content of the Firebase config file or object is considered public.

So I think we can keep this appId public. We just need to put the token via secrets.

Here is [an example](https://devcenter.bitrise.io/en/steps-and-workflows/workflow-recipes-for-android-apps/-android--deploy-to-firebase-app-distribution.html) from Bitrise. They only add `Firebase token` into secrets

Reply:
What do you think?

Reply:
@suho Agree, we don't need to move it into secrets 👍